spec/requests/api/reports_spec.rb

```diff
@@ -76,4 +76,15 @@ def app
     expect_result_to_match_hash(@result, "result_set" => [{"foo" => "bar"}, {"foo" => "baz"}])
     expect_request_success
   end
+
```

Comment:
add a test to verify, user has appropriate role. a 3 liner only. Look at any of the vms tests titled "...without appropriate role"